.github/ISSUE_TEMPLATE/request-for-help.md

```diff
@@ -7,10 +7,24 @@ assignees: ''
 
 ---
 
-Please [search existing issues (open & closed)](https://github.com/weaveworks/eksctl/issues?utf8=%E2%9C%93&q=is%3Aissue) to see if there is a similar one. If there is, add comments or vote where appropriate.
+<!--
+Thank you for opening this issue! You are helping make eksctl a better project :)
```

Comment:
Minor: when someone is asking for help, responding saying they are helping make eksctl a better project, doesn't quite sit right, to me. It may be someone wanting paid support or asking for something out of our scope.